scripts/build_kb.sh

```diff
@@ -1,2 +1,9 @@
-#!/bin/bash
-../sc-machine/bin/sc-builder -f --clear -c ../ostis-web-platform.ini -i ../repo.path -o ../kb.bin
+#!/usr/bin/env bash
+set -eo pipefail
+
+if [ -z "${BINARY_PATH}" ];
```

Comment:
This is strange. We're checking for `BINARY_PATH` but are using `SC_MACHINE_PATH`. It is possible that the former is set, and the latter is not. 
```suggestion
if [ -z "${SC_MACHINE_PATH}" ];
```